tutorials/s4sdk-secure-cloudfoundry/s4sdk-secure-cloudfoundry.md

```diff
@@ -90,168 +69,148 @@ This alternative approach requires that you have npm installed on your machine.
+      env:
+        TENANT_HOST_PATTERN: 'approuter-(.*).cfapps.<REGION_IDENTIFIER>.hana.ondemand.com'
+        destinations: '[{"name":"app-destination", "url" :<APPLICATION_URL>, "forwardAuthToken": true}]'
+      services:
+        - my-xsuaa
+    ```
 
-The **TENANT`_HOST_PATTERN`** is a variable that declares the pattern how multiple tenants in the URL are identified and handled. During runtime, the App Router will match the incoming host against the pattern and tries to extract the regular expression from the pattern. It will use the matched string to delegate authentication to the respective XSUAA tenant (in SAP Cloud Platform the tenant ID corresponds to the subaccount ID. If you desire different URL patterns, you need to change this pattern accordingly.
+    Adapt the file as follows:
 
-Note that the **TENANT`_HOST_PATTERN`** variable is only required in real multi-tenant application, i.e, applications where a physical deployment serves multiple clients from the same deployment. We assume in this blog series that we want to build multi-tenant applications, as we aim towards cloud-native development. However, this variable is not necessary if you have a single-tenant application. To realize this, the **`xs-security`.json** security descriptor may declare tenant-mode: dedicated (see step 5 below).
+      - Just like before provide the values for your account and region
+      - In `destinations` replace `<APPLICATION_URL>` with the actual URL of your previously deployed app.
 
-**Destinations** is a variable that declares the internal routes from the App Router to the underlying backend microservices. As we only have one hello world microservice yet, we define only one destination called app-destination here. This app-destination is referenced by the previously created `xs-app`.json file.
+**Note**: It is mandatory that your app URL conforms the format `<APP_NAME>-<SUBACCOUNT>.cfapps.<REGION_IDENTIFIER>.hana.ondemand.com`. The authentication mechanism will identify you by pulling the user-id (`SUBACCOUNT`) out of the subdomain.
 
-The **services section** declares to bind our own XSUAA service instance to the App Router. This binding will ensure a corresponding VCAP`_SERVICE` entry that holds the client ID, client secret and public key that is required to validate any incoming OAuth token/JWT from the XSUAA service:
+### Understanding the AppRouter's `manifest.yml` and `xs-app.json`
 
-![XSUAA service](Figure3-1-1.png)
+The `TENANT_HOST_PATTERN` is a variable that declares the pattern how multiple tenants in the URL are identified and handled. During runtime, the App Router will match the incoming host against the pattern and tries to extract the regular expression from the pattern. It will use the matched string to delegate authentication to the respective XSUAA tenant (in SAP Cloud Platform the tenant ID corresponds to the subaccount ID. If you desire different URL patterns, you need to change this pattern accordingly.
```

Comment:
While tenant ID = subaccount ID holds, this is not the subdomain used here, so tenant ID is not directly related to the URL pattern.